ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.1.7f1
-m_EditorVersionWithRevision: 2020.1.7f1 (064ffcdb64ad)
+m_EditorVersion: 2021.3.0f1
+m_EditorVersionWithRevision: 2021.3.0f1 (6eacc8284459)
```

Comment:
Just as a general note, ideally we wouldn't use the very first patch version in workflows that demonstrate the workflow works, as they're most prone to having bugs from Unity side. When people clone this project they're more likely to run into errors.

Reply:
We can wait a couple of months to give them time to fix any potential bugs, and then I'll update this PR to use v2021.3.5f1 or something like that.

Reply:
Yea fair point. Feel free to move forward and we can make an issue to update it later.